docs/machine-learning/tutorials/snippets/sentiment-analysis/csharp/SentimentAnalysis.csproj

```diff
@@ -3,10 +3,11 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
   </PropertyGroup>
 
   <PropertyGroup>
-    <LangVersion>7.1</LangVersion>
+    <LangVersion>10.0</LangVersion>
   </PropertyGroup>
```

Comment:
This is the default C# version for the `net6.0` TFM, any reason we're calling it out explicitly?

Reply:
Good question. @jwood803 can you confirm that with the GA version of .NET 6 you don't have to specify it?

Reply:
Removed and the build passes 😄